lib/maths/CBoostedTree.cc

```diff
@@ -1226,14 +1263,18 @@ class CBoostedTree::CImpl final {
         m_Lambda = m_BestHyperparameters.s_Lambda;
         m_Gamma = m_BestHyperparameters.s_Gamma;
         m_Eta = m_BestHyperparameters.s_Eta;
+        m_EtaGrowthRatePerTree = m_BestHyperparameters.s_EtaGrowthRatePerTree;
         m_FeatureBagFraction = m_BestHyperparameters.s_FeatureBagFraction;
         m_FeatureSampleProbabilities = m_BestHyperparameters.s_FeatureSampleProbabilities;
+        LOG_TRACE(<< "lambda* = " << m_Lambda << ", gamma* = " << m_Gamma
+                  << ", eta* = " << m_Eta << ", eta growth rate per tree* = " << m_EtaGrowthRatePerTree
+                  << ", feature bag fraction* = " << m_FeatureBagFraction);
     }
 
     //! Get the number of hyperparameters to tune.
     std::size_t numberHyperparametersToTune() const {
         return (m_LambdaOverride ? 0 : 1) + (m_GammaOverride ? 0 : 1) +
-               (m_EtaOverride ? 0 : 1) + (m_FeatureBagFractionOverride ? 0 : 1);
+               (m_EtaOverride ? 0 : 2) + (m_FeatureBagFractionOverride ? 0 : 1);
```

Comment:
For eta, is the second hyperparam  its growth rate?

Reply:
Yes